pkgs/top-level/python-packages.nix

```diff
@@ -6736,6 +6738,8 @@ in {
 
   tensorly = callPackage ../development/python-modules/tensorly { };
 
+  tensorpack = callPackage ../development/python-modules/tensorpack { };
```

Comment:
This belongs to a different commit, otherwise breaks eval

Reply:
Also, this is still not addressed.

Reply:
fixed!